e2e-polybft/framework/test-cluster.go

```diff
@@ -590,6 +598,12 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 				strings.Join(sliceAddressToSliceString(cluster.Config.BridgeBlockListEnabled), ","))
 		}
 
+		proxyAdminAddr := cluster.Config.ProxyContractsAdmin
+		if proxyAdminAddr == "" {
+			proxyAdminAddr = "0x5aaeb6053f3e94c9b9a09f33669435e7ef1beaed"
```

Comment:
It would be better to put the address of the first validator in cluster as `proxyAdminAddr` if it was not provided by that `WithProxyContractsAdmin`, since we already do this for bunch of other stuff.

Reply:
No, no, you are right. I forgot about that. Then I would say, just move this address to some global variable in test-cluster.